tests/src/core_pdf/impl/spec/s_07_security/test_specification_evidence.py

```
@@ -177,18 +177,84 @@ def test_iso_ts_32003_aes_gcm_rules_match_the_implementation_when_available() ->
+    assert "shall be a direct object" in extension_and_encrypt
+    assert "A 32-byte salt value for use in key derivation" in extension_and_encrypt
+
+    assert "Table 3 — Additions to ISO 32000-2:2020, Table 22" in permission_and_trailer
+    assert "13 When zero, indicates that a PDF MAC token is required" in permission_and_trailer
+    assert "unless bit 13 is zero in all revisions" in permission_and_trailer
+    assert "AuthCode dictionary" in permission_and_trailer
+    assert "Required if the document is encrypted with user access permissions bit 13 zero" in (
+        permission_and_trailer
+    )
+    assert "Shall be a direct object" in permission_and_trailer
+    assert "value of the V entry in the document’s Encrypt dictionary shall be at least 5" in (
```

Ruff (0.16.3)

[warning] String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)